### DISCUSSION TITLE
Add side notes feature for academic writing

### DISCUSSION THREAD

**Discussion by JJger148** [11ty/eleventy#4088] [Enhancement Queue]
### Describe the solution you'd like

I would like to see an easy way to add side notes, so I can write in an academic style. 

https://gwern.net/sidenote

**Comment by rdela** [comment]
@JJger148 did you have any thoughts on “de­sign choices, HTML im­ple­men­ta­tions and their pros/cons”? 

Also I found this article by Siraj Chokshi, [Sidenotes for Markdown](https://sirajchokshi.com/blog/sidenotes-for-markdown)

**Comment by JJger148** [comment]
No particular design choice, although I prefer the simple footnote just like on the first image. 
In regards to markdown format, I use [^1]  for footnotes in Zettlr and Obsidian, it creates the footnote at the bottom like this [^1]: _footnote_ . If this footnote would become a sidetone it would be perfect.

**Comment by rdela** [comment]
Thanks @JJger148 I also ran across these links on Mastodon!

https://mastodon.social/@markllobrera/112327749278630978
https://www.markllobrera.com/posts/sidenotes/

https://mastodon.social/@falcon/111823702861185129

https://social.lol/@robb/112434921466808256

Cc @dirtystylus @anniegreens @rknightuk

**Comment by rdela** [comment]
@JJger148 also https://pack11ty.dev from @nhoizey comes with…

> Enhanced Markdown thanks to an extensive set of Markdown-it plugins for better contribution: footnotes, attributes, headings anchors, abbreviations, containers

Maybe you could try it out or try adding the [Markdown-it footnote plugin](https://github.com/markdown-it/markdown-it-footnote) to your existing setup?

**Comment by rdela** [comment]
But Pack11ty does not appear to me on a casual glance to do side notes so think Mark’s approach is probably the only complete one I know of currently.

**Comment by nhoizey** [comment]
@rdela Pack11ty's default design doesn't put footnotes on the sides, indeed.

**Comment by zachleat** [comment]
> This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.

> View [the enhancement backlog here](https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+). Don’t forget to upvote the top comment with 👍!
